test/assets/preset-mini-targets.ts

```diff
@@ -114,8 +116,6 @@ export const presetMiniTargets: string[] = [
 
   // border - color
   'border-[#124]',
-  'border-[2em]',
-  'border-[calc(1em-1px)]',
```

Comment:
These two are negative cases in which brackets get resolved to color by default

Reply:
I think it's unnecessary.

Reply:
I know, but when I first saw it I thought it was correct until I saw test.snap, I didn't know it could only change colors, which could cause some misunderstanding.

Reply:
Maybe we can refactor to separate all negative cases? The `text-[x]` rule also has similar tests

Reply:
Yes, that's fine. But we should remove it before refactoring.